### PR TITLE
Add Comment story, add spacing above user info

### PR DESCRIPTION
**Pull request by Tirokk**
> [mattwr18](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-23T16:42:34Z" title="Monday, September 23rd 2019, 6:42:34 pm +02:00">Sep 23, 2019</time>_
_Merged <time datetime="2019-09-25T08:48:20Z" title="Wednesday, September 25th 2019, 10:48:20 am +02:00">Sep 25, 2019</time>_
---

### CODE REVIEW

File: webapp/components/Comment/Comment.vue [hunk 1]
```diff
@@ -11,7 +11,7 @@
   </div>
   <div v-else :class="{ comment: true, 'disabled-content': comment.deleted || comment.disabled }">
     <ds-card :id="`commentId-${comment.id}`">
-      <ds-space margin-bottom="small">
+      <ds-space margin-bottom="small" margin-top="base">
```
Comment (Tirokk) [inline webapp/components/Comment/Comment.vue:14]: > [Tirokk](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-09-25T08:19:13Z" title="Wednesday, September 25th 2019, 10:19:13 am +02:00">Sep 25, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -11,7 +11,7 @@
   </div>
   <div v-else :class="{ comment: true, 'disabled-content': comment.deleted || comment.disabled }">
     <ds-card :id="`commentId-${comment.id}`">
-      <ds-space margin-bottom="small">
+      <ds-space margin-bottom="small" margin-top="base">
```


Yeah!

File: webapp/components/CommentForm/CommentForm.vue [hunk 2]
```diff
@@ -123,6 +123,7 @@ export default {
               data: { UpdateComment },
             } = res
             this.$emit('updateComment', UpdateComment)
+            this.$emit('collapse')
```
Comment (Tirokk) [inline webapp/components/CommentForm/CommentForm.vue:126]: > [Tirokk](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-09-25T08:20:23Z" title="Wednesday, September 25th 2019, 10:20:23 am +02:00">Sep 25, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -123,6 +123,7 @@ export default {
               data: { UpdateComment },
             } = res
             this.$emit('updateComment', UpdateComment)
+            this.$emit('collapse')
```


Nice !!! 💫

**Comment by Tirokk** [comment]
> [mattwr18](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-25T08:43:10Z" title="Wednesday, September 25th 2019, 10:43:10 am +02:00">Sep 25, 2019</time>_

---


yeah, I'm not sure why it looks slightly different in storybook... :woman_shrugging: 
the ellipses are deployed to production, and I guess come from using the `truncate` filter....
I think these might be desired though :woman_shrugging:

**Review by Mogge — APPROVED** [review]
Glad @mattwr18 that you changed the ugly top margin 🤗

Great that you add a comment story !!! 💫🚀
Love how show less works now @mattwr18 !!! 😍

Just to make you aware:
In the storybook the context menu is not aligned with the user …
… I can not say why …

Bildschirmfoto 2019-09-25 um 10 28 25

Probably not related to this PR, but there are to ellipses now if the comment is collapsed:

Bildschirmfoto 2019-09-25 um 10 14 17

**Pull request closed without merging**